File: modified_scripts/julia/bubble_sort/gemma_2b.jl
```julia
function bubble_sort(nums)
    for i = 1, length(nums)
        for j = i, length(nums)
            if nums[j] <= nums[j - 1]
                swap(nums, i, j)
            end
        end
        i = i + 1
    end
    return nums
end

function swap(nums, i, j)
    temp = nums[i]
    nums[i] = nums[j]
    nums[j] = temp
end
```

File: modified_scripts/r/bubble_sort/gemma_2b.r

bubble_sort <- function(nums) {
  for (i in 1:length(nums)) {
    for (j in 2:length(nums)) {
      if (nums[j] < nums[j - 1]) {
        temp <- nums[j]
        nums[j] <- nums[j - 1]
        nums[j - 1] <- temp
      }
    }
  }
  return(nums)
}